recipe/gen_patch_json.py

```diff
@@ -1203,8 +1203,9 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         # incompatibilities with markupsafe >=2.1 and we are constraining
         # markupsafe <2 to be on the safe side
         # https://github.com/pallets/jinja/issues/1585
-        if record_name == "jinja2" and record['version'].startswith(
-                ('2.9.', '2.10.', '2.11.')):
+        if record_name == "jinja2" and \
+                (record['version'].startswith(('2.9.', '2.10.', '2.11.')) or 
```

Comment:
can you be more precise about the build number for 2.11.3? You seem to be building it again, so we don't want new builds to get patched again.

Reply:
Sure, that's a good idea.

Reply:
@hmaarrfk, I med your suggested change.  It didn't change the output of `show_diff.py` as far as I can tell but I pasted the new output above in the PR.